src/Hazelcast.Net.Tests/Serialization/DataInputOutputTest.cs

```diff
@@ -23,20 +23,32 @@ namespace Hazelcast.Tests.Serialization
 {
     internal class DataInputOutputTest
     {
-        private static readonly Endianness[] Endiannesses =
+        private static readonly object[] Endiannesses =
```

Comment:
This is not just `Endiannesses` anymore. Maybe we can rename the parameter

Reply:
+1